src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -263,9 +282,11 @@ public void validateArguments() {
 
         // throw if args were specified for a filter that was also disabled
         disableFilters.forEach(s -> {
-            if (requiredPredecessors.contains(s)) {
+            if (requiredPredecessors.contains(s) && !toolDefaultReadFilters.containsKey(s)) {
                 throw new CommandLineException(
                         String.format("Values were supplied for (%s) that is also disabled", s));
```

Comment:
There is a test case for throwing that is disabled because it fails after this change (I think that this is something related with Barclay, I added a `TODO` there for more information) and a new one covering the bug that I reported in #2357 (just logging). Thus, the only case where this could be exercised is when a non-default filter is disabled and arguments provided. Next commit will include that test.